### ISSUE TITLE
Next: responsive images markup and a11y issues, validator w3c

### ISSUE THREAD

**Issue by Fupete** [italia/designers.italia.it#466]
We need to check settings and code we use for the responsive image plugin activated to fix https://github.com/italia/designers.italia.it/issues/411. A validation this morning shows various issues. Here follow the main three, the first one is also an a11y issue. 

1️⃣ The markup by the img responsive plugin contains an `img` tag with `alt=""` together with `role="presentation"`, see for example the main image on the hero. Maybe it is losing the alt from the real image few lines of code below inside the tag picture? Or is the role mistaken? **A11y issue.** 

![issue on markup](https://user-images.githubusercontent.com/6736103/203931809-c9af5216-a05c-463c-9f6d-224137fa45c7.png)

![issue by w3c validator](https://user-images.githubusercontent.com/6736103/203932663-516d9080-2bdc-4873-ba14-4f22034c9047.png)

------

2️⃣ The src contain strange characters ` %3C, ...`, interpreted as bad value from the w3c validator: 
`src="data:image/svg+xml;charset=utf-8,%3Csvg heig…g&#x27; version=&#x27;1.1&#x27;%3E%3C/svg%3E` 

![second issue by w3c validator](https://user-images.githubusercontent.com/6736103/203932622-d2cc2933-0fc7-460d-8e24-b1a057594380.png)

-----

3️⃣ The validator also showed different errors about `srcset` missing (we have a `data-srcset`)... 
![various issues by w3c validator](https://user-images.githubusercontent.com/6736103/203933039-7aa13986-3a33-43d6-ab99-dff69cd9fb14.png)

More details here https://validator.w3.org/nu/?doc=https%3A%2F%2Fdesigners-italia-it-next.vercel.app%2Flevel1%2F

**Comment by Fupete** [comment]
cc @bfabio @astagi @OscarBresolin

**Comment by gpeirolo** [comment]
@Fupete all this issues are generated by the gatsby-image-plugin:

1. this is the svg preview image generated by the plugin it gets hidden as soon as the real image loaded
2. the same image, an svg generated on the fly by the plugin to give a low resolution and blurred preview
3.  this are just lazyload properties that will be substituted with srscet once JS runs

There's not much we can do about the but consider this:
- Gatsby generates static pages for SEO purpose but once loaded the DOM gets manipulated by the client's JS
- All these errors disappear from the DOM or SR once the JS kicks in

More on this: https://github.com/gatsbyjs/gatsby/issues/8256

**Comment by Fupete** [comment]
Tx. I close this and mark it to remember as a note.
